### DISCUSSION TITLE
Chrome v3.3

### DISCUSSION THREAD

**Discussion by rizwansoaib** [rizwansoaib/whatsapp-monitor#226] [General]
#218 #223 #145 #216 Bug Fixed and UI improvements

Added Voice Notification - TrackWapp Alert Your Partner is online in WhatsApp

[Chrome3.3_Final.zip](https://github.com/user-attachments/files/17690477/Chrome3.3_Final.zip)

<hr /><em>This discussion was created from the release <a href='https://github.com/rizwansoaib/whatsapp-monitor/releases/tag/v3.3'>Chrome v3.3 </a>.</em>

**Comment by IzzaYaq** [comment]
I added the ext it keeps on sayig open tracker on contact, which I have done and its not working

**Comment by rizwansoaib** [comment]
After installation of extension 


Open whatsapp web 

It's should start after 30 seconds 

In sidebar you will see CSV downloader as well as icon image button 



Send me logs from extension error and by pressing Ctrl+shift+c then console tab after open whatsapp web

**Comment by IzzaYaq** [comment]
My logs are empty
Sent from my iPhone

On Dec 1, 2024, at 5:44 PM, RIZWAN AHMAD ***@***.***> wrote:

﻿

After installation of extension

Open whatsapp web

It's should start after 30 seconds

In sidebar you will see CSV downloader as well as icon image button

Send me logs from extension error and by pressing Ctrl+shift+c then console tab after open whatsapp web

**Comment by IzzaYaq** [comment]
I try to add the number it keeps on saying use extension first on this contact
Sent from my iPhone

On Dec 1, 2024, at 5:44 PM, RIZWAN AHMAD ***@***.***> wrote:

﻿

After installation of extension

Open whatsapp web

It's should start after 30 seconds

In sidebar you will see CSV downloader as well as icon image button

Send me logs from extension error and by pressing Ctrl+shift+c then console tab after open whatsapp web
